README.md

```diff
@@ -21,6 +21,8 @@ This README is for developers who want to contribute to Prebid.js.  For user-fac
 
 ## Install
 
+    $ git clone --depth=1 https://github.com/prebid/Prebid.js.git
```

Comment:
Please omit the `--depth=1` portion Not a bad idea but standard `git clone` should suffice for an intro section